# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/linux/linux24/arch-ppc-mac-nvram.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---


Erhm - on PPlus I do not want the non-compileable in this config
macintosh/nvram driver ...

  - Rene Rebe <rene@exactcode.de>

--- linux-2.4.25/drivers/macintosh/Makefile.orig	2002-11-29 00:53:13.000000000 +0100
+++ linux-2.4.25/drivers/macintosh/Makefile	2004-04-19 21:51:26.000000000 +0200
@@ -28,7 +28,7 @@
 
 obj-$(CONFIG_PMAC_PBOOK)	+= mediabay.o
 obj-$(CONFIG_MAC_SERIAL)	+= macserial.o
-ifneq ($(CONFIG_MAC),y)
+ifeq ($(CONFIG_PMAC),y)
   obj-$(CONFIG_NVRAM)		+= nvram.o
 endif
 obj-$(CONFIG_MAC_HID)		+= mac_hid.o
